nss/src/scrapeRoomFacilities.ts

```diff
@@ -0,0 +1,26 @@
+import { load } from "cheerio";
+import nssFetch from "./nssFetch";
+import { Facilities, FACILITIES_LIST } from "./types";
+
+export const scrapeRoomFacilities = async (
```

Comment:
Not super sure where to put this but can we also scrape info on:
- Floor (flat vs tiered)
- Seating (movable vs fixed)
- Microphones (lectern, neck, etc.)

All of the above were things that the execs were considering when looking for a room for the Alumni Panel, so definitely something worth supporting.